protocol-configuration/dependencies.ts

```diff
@@ -950,6 +950,9 @@ const dependencies: DependencyMap = {
   },
   rariPool8Lusd: {
     contractDependencies: ['rariPool8LusdPCVDeposit', 'rariPool8Comptroller']
+  },
+  timelock: {
```

Comment:
Not sure I follow. `timelock` is still treated as a dependency because it's passed as an arg in the `fip_79x` scripts. 

Do you mean rename to something like `oldTimelock`?

Reply:
As discussed, updating dependencies